Cargo.lock

```diff
@@ -886,7 +886,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "hecate"
-version = "0.76.0"
+version = "0.77.1"
```

Comment:
@mattciferri Can we update the release call to build the project before pushing with

```
cargo build --release
```

I now we removed this because circle now does the builds, but this version should be incremented.